addons/website/static/src/snippets/s_pricelist_boxed/options.js

```diff
@@ -0,0 +1,57 @@
+/** @odoo-module **/
+
+import { _t } from "@web/core/l10n/translation";
+import options from "@web_editor/js/editor/snippets.options";
+
+options.registry.PricelistBoxed = options.Class.extend({
+
+    //--------------------------------------------------------------------------
+    // Options
+    //--------------------------------------------------------------------------
+
+    /**
+     * Show/hide descriptions.
+     */
+    toggleDescription: function (previewMode, widgetValue, params) {
+        const dishes = this.$target[0].querySelectorAll('.s_pricelist_boxed_item');
```

Comment:
And we are supposed to use double-quotes for all strings on the JS side now (unless the string contains a double-quote).